dev_guide/projects.adoc

```diff
@@ -194,3 +194,98 @@ To delete a project using the CLI:
+the {product-title} Pro subscriber) that they are granted access to. They can
+not add projects of their own ownership. Collaborators can only add resources.
+
+[[project-collaboration-adding-collaborators]]
+=== Adding Collaborators
+{product-title} Pro subscribers can add collaborators by following these steps:
+
+. Each user you want to add as a collaborator must create a free account at
+ link:https://developers.redhat.com/[developers.redhat.com]. Once your
+ collaborator has confirmed their Red Hat Developers account, you can add them
+ to your subscription.
+
+ . Each collaborator must sign into
+link:https://developers.redhat.com/[developers.redhat.com] and click on their
+name in the upper-right corner to access their account details. Make note of the
+*Red Hat Login ID* on this page, as it is the user name you will be required to
+enter in order to associate the collaborator with your subscription.
+
+. Sign in to link:https://manage.openshift.com[manage.openshift.com] and click on
+*Manage Subscription* under the cluster you want to add the collaborator to.
++
+image::manage-subscriptions-collaboration.png[]
+
+. Once you are in the subscription management console, click the *Manage* link
+under the *Collaborators* heading, which brings you to the *Collaboration* page.
++
+image::manage-collaborators.png[]
+
+. On the *Collaboration* page, enter the *Red Hat Login ID* for the user in the user
+name field and click *Add Collaborator*.
++
+You should now see the user listed under your collaborators, the time the user
+was added, and an option to remove the user from your subscription.
++
+This does not automatically grant the user any access to your projects. Access
+must be granted manually by the project owner using `oc policy` commands. For
+example, you can log in to the cluster through the CLI using your access token
+and use `oc policy add-role-to-user` to give the user a role using the same user
+name listed on the *Collaboration* page:
++
+----
+~$ oc login https://api.openshift.com --token=<...>
+Logged into "https://api.openshift.com:443" as "exampleuser" using the token provided.
+
+You have one project on this server: "exampleuser-collab"
+
+Using project "exampleuser-collab".
+~$ oc policy add-role-to-user view collaborator-1234
+role "view" added: "collaborator-1234"
+----
++
+This example grants `view` access to the project for user `collaborator-1234`.
+See
+xref:../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Authorization]
+for more information on access roles.
++
+Now, when the collaborator user signs in to
+link:https://manage.openshift.com[manage.openshift.com], they will see a card to
+log in to the web console for the same cluster as your subscription and, if the
+collaborator was granted access to your projects on the cluster, they will have
+access just like any other user.
+
+[[project-collaboration-removing-collaborators]]
+=== Removing Collaborators
+
```

Comment:
@damemi @abhgupta @thrasher-redhat Perhaps more can be said about how to remove collaborators? What steps would the user follow specifically?

Reply:
In the registration application, when you see the list of collaborators you have added, you will see a "Remove" buttton next to each collaborator. Clicking the button will result in the collaborator being removed from your Pro tier subscription. The Pro plan owner is still responsible for removing any roles granted to the collaborator in their projects.

Reply:
You have covered the details adequately (in my opinion) in your section below. Specifically, the Pro tier owner needs to revoke/remove any roles granted to the collaborator in their projects. Additionally, they should remove the user as a collaborator in the registration application.